token-metadata/program/src/processor/burn/burn.rs

```diff
@@ -49,7 +49,6 @@ pub fn burn<'a>(
 
 // V1 implementation of the burn instruction.
 fn burn_v1(program_id: &Pubkey, ctx: Context<Burn>, args: BurnArgs) -> ProgramResult {
-    msg!("Burn V1");
```

Comment:
Can we wait to see if we for sure need the binary space before removing this? These instruction name logs are useful for debugging.